src/components/Router/MainRouter.tsx

```diff
@@ -33,9 +34,12 @@ const WrappedError = withTheme(GenericError)
 type State = {
   crossDeviceInitialClientStep?: number
   crossDeviceInitialStep?: number
+  workflowSteps?: StepConfig[]
 }
 
 export default class MainRouter extends Component<InternalRouterProps, State> {
+  useWorkflowRun = () => !!this.props.options.useWorkflow
```

Comment:
I'm wondering where `useWorkflow` is set and if it would be better to check, if we check for having a workflowRunId on the options.